win32/test/test_sspi.py

```diff
@@ -170,10 +170,10 @@ def testSignKerberos(self):
     def _testSequenceSign(self):
         # Only Kerberos supports sequence detection.
         sspiclient, sspiserver = self._doAuth("Kerberos")
-        key = sspiclient.sign("hello")
-        sspiclient.sign("hello")
+        key = sspiclient.sign(str2bytes("hello"))
```

Comment:
trivial matter, but I'd probably be a little happier with `b"hello"` here - `str2bytes` etc were really just about supporting py2 and py3 and also managing the transition, but ultimately I'd like to see them die.